includes/dev-box-limits.md

```diff
@@ -0,0 +1,19 @@
+---
+author: dhruvmu
+ms.service: dev-box
+ms.topic: include
+ms.date: 09/29/2022
+ms.author: dhruvmu
+---
+
+| Subscription Type                                                             | VM Cores  |
+|-------------------------------------------------------------------------------|-----------|
+| Pay as you go<br>MSDN devtest<br>Sponsored <br>AzurePass<br>CSP<br>CSPdevtest | 20        |
```

Comment:
I've been working on this same article for the Azure Lab Services team, and we decided to remove references to internal only subscriptions, including CSPDevTest and MSDNDevTest. What are your thoughts, given this information is public facing?